docs/Ecosystem.md

```diff
@@ -153,6 +153,7 @@ Plugins maintained by the Fastify team are listed under [Core](#core) while plug
 - [`fastify-reverse-routes`](https://github.com/dimonnwc3/fastify-reverse-routes) Fastify reverse routes plugin, allows to defined named routes and build path using name and parameters.
 - [`fastify-rob-config`](https://github.com/jeromemacias/fastify-rob-config) Fastify Rob-Config integration.
 - [`fastify-schema-constraint`](https://github.com/Eomm/fastify-schema-constraint) Choose the JSON schema to use based on request parameters.
+- [`fastify-schema-to-typescript`](https://github.com/thomasthiebaud/fastify-schema-to-typescript) Generate typescript types based on your validation schemas so they are always in sync.
```

Comment:
I took a look and what do you think to comment regarding that these schemas are generated from a `yaml`

Reply:
Good point, I'm adding something